config/account.go

```diff
@@ -165,7 +165,8 @@ func (a *AccountGDPR) PurposeEnforced(purpose consentconstants.Purpose) (value,
 // PurposeEnforcementAlgo checks the purpose enforcement algo for a given purpose by first
 // looking at the account settings, and if not set there, defaulting to the host configuration.
 func (a *AccountGDPR) PurposeEnforcementAlgo(purpose consentconstants.Purpose) (value TCF2EnforcementAlgo, exists bool) {
```

Comment:
For my own understanding, why was the named return parameter for `TCF2EnforcementAlgo` unchanged here, but in `config/config.go,` the named return parameter for `TCF2EnforcementAlgo` was deleted?

Reply:
Because I made a mistake Alex. I meant to rename "value" with "enforcement" to make it more readable. Modified in 80909c63773e63d36828e243eae563bc04972a4c